konan_sdk/sdk.py

```diff
@@ -42,3 +43,26 @@ def predict(self, deployment_uuid: str, input_data: Union[Dict, str]) -> Tuple[s
         prediction_uuid, output = PredictionEndpoint(api_url=self.api_url, user=self.auth.user, deployment_uuid=deployment_uuid).post(payload=input_data)
 
         return prediction_uuid, output
+
+    def evalute(self, deployment_uuid: str, start_time: datetime.datetime, end_time: datetime.datetime) -> Dict:
+        """Call the evaluate function for a given deployment
```

Comment:
Hmm while this doc string format looks cool, I think we need to be consistent in our SDK documentation, the above endpoints have a different style of formatting than this one, might want to reformat them to match this

Reply:
Used sphinx format like others